Source/Diagnostics/WarpXOpenPMD.H

```diff
@@ -156,7 +159,7 @@ private:
    * @param[in] write_int_comp The int attribute ids, from WarpX
    * @param[in] int_comp_names The int attribute names, from WarpX
    */
-  void SavePlotFile(const std::unique_ptr<WarpXParticleContainer>& pc,
+  void DumpToFile(WarpXParticleContainer* pc,
```

Comment:
is the commented out code on Line 173 needed? it was not part of this PR, so may be can done somewhere else. But just wanted to note it here.